sdk/internal/errinfo/errinfo.go

```diff
@@ -0,0 +1,15 @@
+// +build go1.13
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package errinfo
```

Comment:
Open to other package names.  `errors` would be ideal but collides with the same from the standard library.

Reply:
I don't love `err`.  I would avoid the appreviation on the package name `errorinfo`

Reply:
Agreed, will fix.